packages/@cdktf/hcl2cdk/test/__snapshots__/resources.test.ts.snap

```diff
@@ -67,49 +67,59 @@ new aws.cloudfrontDistribution.CloudfrontDistribution(this, \\"s3_distribution\\
   },
   orderedCacheBehavior: [
     {
-      allowedMethods: [\\"GET\\", \\"HEAD\\", \\"OPTIONS\\"],
-      cachedMethods: [\\"GET\\", \\"HEAD\\", \\"OPTIONS\\"],
+      allowed_methods: [\\"GET\\", \\"HEAD\\", \\"OPTIONS\\"],
+      cached_methods: [\\"GET\\", \\"HEAD\\", \\"OPTIONS\\"],
       compress: true,
-      defaultTtl: 86400,
-      forwardedValues: {
-        cookies: {
-          forward: \\"none\\",
+      default_ttl: 86400,
+      forwarded_values: [
+        {
+          cookies: [
+            {
+              forward: \\"none\\",
+            },
+          ],
+          headers: [\\"Origin\\"],
+          query_string: false,
         },
-        headers: [\\"Origin\\"],
-        queryString: false,
-      },
-      maxTtl: 31536000,
-      minTtl: 0,
-      pathPattern: \\"/content/immutable/*\\",
-      targetOriginId: \\"local_s3_origin_id\\",
-      viewerProtocolPolicy: \\"redirect-to-https\\",
+      ],
+      max_ttl: 31536000,
+      min_ttl: 0,
+      path_pattern: \\"/content/immutable/*\\",
+      target_origin_id: \\"local_s3_origin_id\\",
+      viewer_protocol_policy: \\"redirect-to-https\\",
     },
     {
-      allowedMethods: [\\"GET\\", \\"HEAD\\", \\"OPTIONS\\"],
-      cachedMethods: [\\"GET\\", \\"HEAD\\"],
+      allowed_methods: [\\"GET\\", \\"HEAD\\", \\"OPTIONS\\"],
+      cached_methods: [\\"GET\\", \\"HEAD\\"],
       compress: true,
-      defaultTtl: 3600,
-      forwardedValues: {
-        cookies: {
-          forward: \\"none\\",
+      default_ttl: 3600,
+      forwarded_values: [
+        {
+          cookies: [
+            {
+              forward: \\"none\\",
+            },
+          ],
+          query_string: false,
         },
-        queryString: false,
-      },
-      maxTtl: 86400,
-      minTtl: 0,
-      pathPattern: \\"/content/*\\",
-      targetOriginId: \\"local_s3_origin_id\\",
-      viewerProtocolPolicy: \\"redirect-to-https\\",
+      ],
+      max_ttl: 86400,
+      min_ttl: 0,
+      path_pattern: \\"/content/*\\",
+      target_origin_id: \\"local_s3_origin_id\\",
+      viewer_protocol_policy: \\"redirect-to-https\\",
     },
   ],
   origin: [
     {
-      domainName: \\"aws_s3_bucket.b.bucket_regional_domain_name\\",
-      originId: \\"local_s3_origin_id\\",
-      s3OriginConfig: {
-        originAccessIdentity:
-          \\"origin-access-identity/cloudfront/ABCDEFG1234567\\",
-      },
+      domain_name: \\"aws_s3_bucket.b.bucket_regional_domain_name\\",
+      origin_id: \\"local_s3_origin_id\\",
```

Comment:
This has been there before, but it seems like the value is an unreplaced reference